components/spin/index.tsx

```diff
@@ -115,18 +117,20 @@ export default class Spin extends React.Component<SpinProps, any> {
         [`${prefixCls}-blur`]: spinning,
       });
       return (
-        <Animate
-          {...divProps}
-          component="div"
-          className={`${prefixCls}-nested-loading`}
-          style={null}
-          transitionName="fade"
-        >
-          {spinning && <div key="loading">{spinElement}</div>}
-          <div className={containerClassName} key="container">
-            {this.props.children}
-          </div>
-        </Animate>
+        <div className={wrapperClassName}>
+          <Animate
+            {...divProps}
+            component="div"
+            className={`${prefixCls}-nested-loading`}
```

Comment:
What I mean, instead of:
```
        <div className={wrapperClassName}>
          <Animate
            {...divProps}
            component="div"
            className={`${prefixCls}-nested-loading`}
            style={null}
            transitionName="fade"
          >
            {spinning && <div key="loading">{spinElement}</div>}
            <div className={containerClassName} key="container">
              {this.props.children}
            </div>
          </Animate>
        </div>
```

I must write:
```
          <Animate
            {...divProps}
            component="div"
            className={`${prefixCls}-nested-loading ${wrapperClassname}`}
            style={null}
            transitionName="fade"
          >
            {spinning && <div key="loading">{spinElement}</div>}
            <div className={containerClassName} key="container">
              {this.props.children}
            </div>
          </Animate>
```
Sorry. Maybe I missunderstanding something :cry: